src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -67,115 +67,108 @@ export function uiRenderMixin(kbnServer, server, config) {
     },
   });
 
-  // register the bootstrap.js route after plugins are initialized so that we can
-  // detect if any default auth strategies were registered
-  kbnServer.afterPluginsInit(() => {
-    const authEnabled = !!server.auth.settings.default;
```

Comment:
Long diff in this file, but I only removed this `kbnServer.afterPluginsInit` call. The code inside it did not change.